hs_core/views/__init__.py

```diff
@@ -994,7 +1002,7 @@ def undo_share_resource_with_user(request, shortkey, user_id, *args, **kwargs):
         if undo_user_privilege == PrivilegeCodes.VIEW:
             undo_user_privilege = "view"
```

Comment:
I know it's not your code, but was === cleaner/more appropriate in these conditionals?

Reply:
Using `===`  just means it will also check for the type. So, yes. It is almost always better to use it.

Reply:
Oh, '===' are used in javaScript, so these ones do not to be updated.

Reply:
Yes, I was lost switching back and forth in the code review. GH view always messes with me.

Reply:
And earlier today I was at lunch on my phone, reading GH screens ....